### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Bupilipili**
### In this branch.
-Updated the features file with background images.
-Updated the index.html file.
-Updated the style.css file.

### CODE REVIEW

File: style.css
Comment (IngDavidTM) [inline style.css:1]: - [x] Although your portfolio displays excellent styling :+1:, it is necessary for your project to be fully responsive up to the desktop version, as per the project requirements. Presently, your app appears visually appealing on larger screens, but it lacks responsiveness between mobile and desktop sizes. Users are forced to utilize horizontal scrolling to view all the elements, which is undesirable. Therefore, it is crucial to ensure that your app adapts seamlessly to different screen sizes in order to provide a smooth user experience :+1:.


https://github.com/Bupilipili/MY-PORTFOLIO/assets/105117832/5f26dab7-d0fd-4f9f-9dc9-baa37e35bed9

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Bupilipili  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bupilipili** [comment]
The webhint test could not be fixed...

**Review by jerryowusu — APPROVED** [review]
Hi @Bupilipili  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Neat and well organized code :heavy_check_mark: 
Great effort in making the requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jerryowusu** [comment]
> The webhint test could not be fixed...

Do not worry about that for now since it is a general problem

**Pull request merged**